lib/rantly/generator.rb

```diff
@@ -158,7 +158,13 @@ def float(distribution=nil, params={})
   def range(lo=nil,hi=nil)
     lo ||= INTEGER_MIN
     hi ||= INTEGER_MAX
-    rand(hi+1-lo) + lo
+
+    if lo.is_a?(Integer) && hi.is_a?(Integer)
```

Comment:
I think a better solution would be using a range in rand: `rand(lo..hi)` :wink:

Reply:
Strangely `rand(lo..hi)` doesn't work. I don't know why it was written this way originally, but I've left it as it is.

Reply:
Introduced in commit 15a4eb0d.

Reply:
@Stratus3D what do you exactly mean with _it doesn't work_? :confused:

Reply:
@Stratus3D I think you didn't see my comment, what do you exactly mean with it doesn't work? can you please elaborate?